src/status_im/ui/screens/home/views.cljs

```diff
@@ -178,14 +279,65 @@
         [react/view {:style               styles/counter-public
                      :accessibility-label :notifications-unread-badge}]])]))
 
+(views/defview qr-button []
+  [react/view {:style {:margin-left 12}}
+   [quo2.button/button {:style {:width 32 :height 32 :background-color (quo2.colors/theme-colors quo2.colors/ui-background-03-light quo2.colors/ui-background-03-dark) :border-radius 10}
+                        :accessibility-label "qr-button"
+                        :size 32
+                        :on-press #(do
+                                     (re-frame/dispatch [::qr-scanner/scan-code
+                                                         {:handler ::qr-scanner/on-scan-success}]))}
+    [icons/icon :main-icons/qr2 {:color (quo2.colors/theme-colors quo2.colors/black quo2.colors/white)}]]])
+
+(views/defview scan-button []
+  [react/view
+   [quo2.button/button {:style {:width 32 :height 32 :background-color (quo2.colors/theme-colors quo2.colors/ui-background-03-light quo2.colors/ui-background-03-dark) :border-radius 10}
```

Comment:
same here
1) why defview ?
2) extra view
3) use type for button instead of style